### ISSUE TITLE
Glass Jar Whacks

### ISSUE THREAD

**Issue by Geevies** [Aurorastation/Aurora.3#11714]
Trying to pick up a greimorian larva with a glass jar will cause you to attack them. I think the jar might need the NOBLUDGEON flag.

**Comment by Doxxmedearly** [comment]
Has the flag, but confirmed this. So much for praying for an easy fix...

**Comment by Doxxmedearly** [comment]
Caused by trying to scoop it with an open lid, since that's a totally different item path. Funny enough, you can scoop them fine with a lid. This is going to probably need us to refactor how glass jars work, probably in a way that's standard for containers that can hold both reagents and items.

**Comment by Batrachophreno** [comment]
Yeah this is going to be part of reagent_container rework. Sadly. At least it's planned, though no ETA.
